os/etc/service/background_tasks/run

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /etc/adcmenv
+
+waitforwsgi
+
+echo "Run process_tasks ..."
+python "${adcmroot}/python/manage.py process_tasks --log-std"
```

Comment:
>> --log-std

It should be log file, not stdout

Reply:
Yes, see to doc string, --log-std Redirect stdout and stderr to the logging system